packages/synapse-react-client/src/utils/functions/SanitizeHtmlUtils.ts

```diff
@@ -1,4 +1,4 @@
-import { IFilterXSSOptions } from 'xss'
+import { escapeAttrValue, IFilterXSSOptions, safeAttrValue } from 'xss'
 
 // PORTALS-1450: including 'style' in the allow-list will cause string values to come through, which crashes the app when used (because it uses jsx).
 export const xssOptions: IFilterXSSOptions = {
```

Comment:
I'll make a ticket to expose this and have just one single XSS sanitization method